websocket/2.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: "Jakarta WebSocket 2.0 (under development)"
-date: 2020-01-15
+title: "Jakarta WebSocket 2.0 (RC1)"
```

Comment:
I think most of the specs call it (milestone)

Reply:
Updated